File: 프로그래머스/lv1/12912. 두 정수 사이의 합/두 정수 사이의 합.py
def solution(a, b):
    # answer = 0
    # if(a <= b):
    #     for i in range(a,b+1):
    #         answer += i
    # else:
    #     for i in range(b,a+1):
    #         answer += i
    # return answer
    
    # range로 범위 지정
    # return sum(range(min(a, b), max(a, b)+1))

    # tuple을 이용한 전환
    if a > b:
        a, b = b, a
    return sum(range(a, b + 1))

File: 프로그래머스/lv1/12912. 두 정수 사이의 합/두 정수 사이의 합.java
class Solution {
    public long solution(int a, int b) {
        long answer = 0;
        
        // (b - a + 1) * (a + b) / 2 -> 등차수열 공식
        for(int i=Math.min(a,b);i<=Math.max(a,b);i++){
            answer += i;
        }
        
        return answer;
    }
}